index.html

```diff
@@ -158,7 +158,7 @@
 
                         this.tracks.push(newTrack({
                             title: this.newTrackTitle,
-                            number: number,
+                            number: number.replace(/^0+/, ''),
```

Comment:
This is incorrect. You will need to update parse.js instead. Also, you will need to provide a unit test to prevent regressions in the future - see parse.test.js.

Reply:
It's not incorrect.

Reply:
I have submitted a new pull request that should satisfy requirements.

Reply:
It's incorrect because this would prevent a track number from being able to start with zero, which includes if the user wants to enter it manually that way. Whereas we just want the parser to assume that the leading zeros are not necessary.